src/constant.rs

```diff
@@ -146,4 +161,13 @@ mod tests {
         assert_eq!(Some(Duration::from_secs(1)), exp.next());
         assert_eq!(None, exp.next());
     }
+
+    #[test]
+    fn test_constant_with_jitter() {
+        let mut it = ConstantBuilder::default().with_jitter().build();
+
+        let dur = it.next().unwrap();
+        fastrand::seed(7);
+        assert!(dur > Duration::from_secs(1));
```

Comment:
It's hard to test random things, this should be something like `delay <= dur < delay + rng`

Reply:
Are you willing to start a new PR for this?

Reply:
Maybe for unit test is enough. I will try to find a better solution, but I'm not sure.